src/ui/wxWidgets/PWSgrid.h

```diff
@@ -54,7 +54,10 @@ typedef std::map<pws_os::CUUID, int, std::less<pws_os::CUUID> > UUIDRowMapT;
  */
 
 class PWSGrid: public wxGrid
-{    
+{
+  typedef std::multimap<wxString, const CItemData*, std::greater<wxString> > DescendingSortedCollection;
+  typedef std::multimap<wxString, const CItemData*, std::less<wxString> >    AscendingSortedCollection;
```

Comment:
Maybe rename these two to `...Multimap` since they are a specific type of collection?